README.md

```diff
@@ -2,163 +2,105 @@
 
-
 Application to transfer data from heterogeneous research data repositories into Open OnDemand. With the possibility of syncing the data back.
 
 Creating the perfect loop for research data
 
-The first repository integration is for Dataverse.
+The first remote repository integration is Dataverse.
 Dataverse will be used as the reference implementation for all features related to this application.
 
-## 🚀 Getting Started
+OnDemand Loop has been designed from the ground up to be a companion to the Open OnDemand application.
+It is deployed into the Open OnDemand environment as another passenger app.
+It has been configured to be deployed under /var/www/ood/apps/sys/loop
+
+The OnDemand environment will launch the application in the same way as the Dashboard app.
+The URL will be https://<ood-server>/pun/sys/loop
+
+## 🚀 Deployment
+The application has been designed to be installed as a OnDemand passenger app.
+
+### System Requirements
+- `Ruby`
+- `NodeJS`
+
+The application has been built to work with the same Ruby and NodeJS requirements as Open OnDemand.
+
+### Installation
+Select a version to deploy, we recommend to install the [latest version ](https://github.com/IQSS/ondemand-loop/releases/latest)
+To install the application as a system application, checkout the release into the OnDemand server sys folder and build the application:
+
+```bash
+cd /var/www/ood/apps/sys
+git clone --branch v1.0.0#2025-06-06 --depth 1 https://github.com/IQSS/ondemand-loop.git loop
+cd loop
+./loop_build.sh
+```
+
+The build process simply install the dependencies into the `vendor/bundle` application folder and compile the CSS and Javascript files.
+
+Now launch the application: `https://<your-server>/pun/sys/loop`
+
+## 🛠️ Development
+To try the application locally or to update or create new features,
+we have created a Docker based local environment to build and run the application.
+
+The required Docker images to build and run has been created and uploaded into the official DockerHub:
+ - [App Builder Images](https://hub.docker.com/r/hmdc/ondemand-loop/tags)
+ - [OnDemand Environment Images](https://hub.docker.com/r/hmdc/sid-ood/tags)
+
+The versions used are referenced in the [Makefile file](./Makefile)
 
 ### Prerequisites
 Ensure you have the following installed:
 - [Docker](https://www.docker.com/)
 - [Docker Compose](https://docs.docker.com/compose/)
 - `make` (usually pre-installed on Linux/macOS, Windows users may need to install it via WSL or Git Bash)
 
+### Local Environment
+The local environment uses Docker compose to start the following containers:
+ - Open OnDemand v3.1.7
+ - SMTP server
 
-## 📦 Available commands
+Docker compose will mount the local `application` directory into the OOD `/var/www/ood/apps/sys/loop` folder.
 
-The following `make` commands are available to manage the application:
-
-### Build and Install Dependencies
-- **Build the Docker container to build the application:**
-  ```sh
-  make loop_docker_builder
-  ```
```

Comment:
don't we need to create the docker builder first anymore?

Reply:
No - the docker image is in DockerHub